docs/user-guide/zonal-average.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<p>Displaying one of those 6 lat bands on the global map would be a great visual help for the reader, if easily possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>